modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -4,15 +4,37 @@
  */
 
 import {API} from '@liferay/object-js-components-web';
-import {openModal, sub} from 'frontend-js-web';
+import {createResourceURL, openModal, sub} from 'frontend-js-web';
 import {SetStateAction} from 'react';
 
 import {formatActionURL} from '../../utils/fds';
 import {
 	firstLetterUppercase,
 	removeAllSpecialCharacters,
 } from '../../utils/string';
-import {ViewObjectDefinitionsModals} from './ViewObjectDefinitions';
+import {DropDownItems} from '../ModelBuilder/types';
+import {
+	DeletedObjectDefinition,
+	ViewObjectDefinitionsModals,
+} from './ViewObjectDefinitions';
+
+type definitionNodeActionsProps = {
```

Comment:
@igor-franca I think we following the pattern of capitalizing the name of types and interfaces.

Reply:
Hey @naraandrade, the other types in this file have the default to start with lowercase letters, I will make the changes.